aws-concepts/aws-ses.md

```diff
@@ -0,0 +1,98 @@
+---
+title: AWS SES (Simple Email Service) | BlueFox Email
+description: Amazon SES is a cloud email service provider that helps digital marketers and application developers send marketing, notification, and transactional emails.
+thumbnail: /assets/glossary/aws-concepts-glossary.png
+sidebar: false
+layout: post
+category: glossary
+
+head:
+  - - meta
+    - name: description
+      content: Amazon SES is a cloud email service provider that helps digital marketers and application developers send marketing, notification, and transactional emails.
+  - - meta
+    - property: og:title
+      content: AWS SES (Simple Email Service) | BlueFox Email
+  - - meta
+    - property: og:description
+      content: Amazon SES is a cloud email service provider that helps digital marketers and application developers send marketing, notification, and transactional emails.
+  - - meta
+    - property: og:image
+      content: https://bluefox.email/assets/glossary/aws-concepts-glossary.png
+  - - meta
+    - property: og:url
+      content: https://bluefox.email/aws-concepts/aws-ses
+  - - meta
+    - name: twitter:card
+      content: summary_large_image
+  - - meta
+    - name: twitter:title
+      content: AWS SES (Simple Email Service) | BlueFox Email
+  - - meta
+    - name: twitter:description
+      content: Amazon SES is a cloud email service provider that helps digital marketers and application developers send marketing, notification, and transactional emails.
+---
+
+# AWS SES (Simple Email Service)
```

Comment:
here as well... in headings, use Amazon SES, in paragraphs, feel free to use Amazon SES or AWS SES.

Reply:
or just SES